README.md

```diff
@@ -1,6 +1,34 @@
 # hostmgr
 
-A description of this package.
+## What is it?
+
+This tool suite is used to create and manage VMs on our Buildkite CI Mac hosts:
+
+ - `hostmgr` is a command-line tool to:
+    - Create, package and publish VMs… (when we prepare new VMs for new macOS or Xcode versions)
+    - List installed VMs, fetch VMs, start and stop VMs… (when running this tool on our CI hosts, to boot VMs and run Buildkite jobs in them)
+ - `hostmgr-helper` is a helper tool used to display the VM's GUI in a window on the host machine.
+    - This helper shows up as a Menu Bar item in your Mac
+    - It is especially useful when setting up new OS templates, to go through the macOS setup wizard via the GUI when configuring the template.
```

Comment:
Perhaps giving a bit more context of what are "new OS templates" will make it easier to understand?

Reply:
Should we just cross-link to the buildkite-ci README in arc/agents/macos-vms/ for more info on that part?

Reply:
```suggestion
    - It is especially useful when [creating new OS templates for VMs](https://github.com/Automattic/buildkite-ci/blob/trunk/src/agents/macos-vms/README.md), to go through the macOS setup wizard via the GUI when configuring the template VM.
```

Reply:
That's a nice idea, thanks!